pyaedt/modules/Mesh.py

```diff
@@ -382,9 +382,10 @@ def assign_surface_mesh_manual(self, names, surf_dev=None, normal_dev=None, aspe
         aspect_ratio_enable = 2
 
         if not surf_dev:
+            surf_dev_enable = 0
```

Comment:
@maxcapodi78 I added some unit tests for the default value while "playing" with this feature and others.

Reply:
great thanks